README.md

```diff
@@ -122,8 +135,14 @@ RUN dpkg --add-architecture arm64 && \
     apt-get install --assume-yes libfoo:arm64
 ```
 
-```
-$ docker build -t my/image:tag path/to/where/the/Dockerfile/resides
+#### Pre-build hook
+
+`cross` enables you to add dependencies and run other necessary commands in the image before using it.
+This action will be added to the used image, so it won't be run every time you use `cross`.
+
+``` toml
+[target.x86_64-unknown-linux-gnu]
+pre-build = ["yum update", "yum install -y libfoo"]
 ```
```

Comment:
Maybe this shouldn't be called `pre-build`.

My concern is that someone would actually like something like what #635 does, e.g run everytime, and this doesn't allow that.

However, that's kinda what `build.rs` is for